EIPS/eip-5247.md

```diff
@@ -0,0 +1,82 @@
+---
+eip: 5247
+title: Smart Proposal
+description: An interface of "proposals" that is submitted to, recorded on and possibly executed and enforced onchain.
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-5247-executable-proposal-standard/9938
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-13
+requires: 7
```

Comment:
Thank you for the suggestion. but I would humbly reject this revision suggestion
I was hoping to have a separate `reference eip` or `related eip` in the preemble. But while it's still `requires`, I would not add 1202 to avoid confusion. The purpose is to decouple 1202 and 5247

Reply:
Let me know if it's ok for you @Pandapip1

Reply:
It's a bit of a contentious point, so I'll let this slide for now. It can always be changed later if/when this goes to Review, for example.

Reply:
Can you rewrite the text of the EIP so it doesn't reference EIPs that aren't required?  Generally speaking, EIPs shouldn't be making comparative arguments, they should focus exclusively on specifying a thing.  If you are just referring to other EIPs to compare against then that sort of content probably should just be left out of the EIP entirely.